### ISSUE TITLE
Better Dense Obsidian Recepe

### ISSUE THREAD

**Issue by Pinaz9** [McJty/DeepResonance#212]
As it is, dense obsidian is a real pain to get. If this isn't intentional, perhaps a new recipe could be made for it, one that uses up RCL or resonating plates instead of requiring spent filter material, which can only be obtained by filtering RCL.

**Comment by McJty** [comment]
I usually have MUCH more spent filter material then I need to make dense obsidian. Not sure exactly why this alternative would be needed?

**Comment by Pinaz9** [comment]
Maybe I'm wrong about it then. I was under the impression that it only takes a stack or two of filters to produce a max purty crystal, but again, I could be wrong. How many does it usually take?

**Comment by McJty** [comment]
Well not sure but I usually don't stop at purifying a single crystal. I don't know exaclty how much purifier you need though but I know that on my LP I have a chest full of spent filter material ;-)

**Comment by Pinaz9** [comment]
I guess I'm used to putting 30 in at a time, and not doing another batch until I need another crystal, as items are easier to manage quantitatively than fluids. Though, this does give me an idea for a build. Tell me, if you point lasers at multiple sides of the same tank, will all the lasers have an effect?

**Comment by McJty** [comment]
yes they will. Keep the purity in mind though. Don't let it drop to 0 or liquid will be destroyed

**Comment by Pinaz9** [comment]
Right. I was thinking that if I have like five of them, I could feed a purifier (diamond, emerald ect.) to one laser, and ghast tears and blaze powder to all the others. Thanks for the help!
